dcos/dcos-ui
# [spike] DCOS-39292: Edit Jobs: Make ID field non - editable

Disable the job ID when editing the job
for better user experience.

Closes https://jira.mesosphere.com/browse/DCOS-39292

## Testing
1. Go to Jobs tab.
2. Create a new job and verify that the job ID field is not disabled.
3. Edit that job and verify that the job ID field is disabled.
4. Verify that the added test makes sense.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
### Before
![Screenshot from 2019-05-22 11-13-12](https://user-images.githubusercontent.com/40791275/58158190-9b9f1e80-7c82-11e9-8f87-bf1ebec2842e.png)

### After
![Screenshot from 2019-05-22 10-52-38](https://user-images.githubusercontent.com/40791275/58158165-8aeea880-7c82-11e9-899a-565811523911.png)
